include/links-abstract-domain.php

```diff
@@ -67,8 +68,15 @@ protected function set_home_url( $language ) {
 	 * @return string The modified url.
 	 */
 	public function site_url( $url ) {
+		if ( ! $this->model->are_language_objects_created() ) {
```

Comment:
Another attempt ;)
Would `if ( false === $this->cache->get( 'languages' ) )` work?

Reply:
As we saw, sometimes the cache is cleaned before passing through it...